modules/accounting/includes/classes/AccountingTutorial.php

```diff
@@ -0,0 +1,343 @@
+                            'next_trigger' => [],
+                            'options'      => [
+                                'content'  => '<h3>' . esc_html__( 'Add Opening Balance', 'erp' ) . '</h3>' .
+                                                '<p>' . esc_html__( 'Add your business opening balance & add initial balance in your Bank accounts.', 'erp' ) . '</p>',
+                                'position' => [ 'edge'  => 'top', 'align' => 'left' ]
+                            ],
+                        ],
+                        'transaction' => [
+                            'target'       => '#erp-accounting #erp-act-menu-transactions',
+                            'next'         => '',
+                            'next_url'     => admin_url( $this->base_url . 'tab=journal#transactions/journals' ),
+                            'next_trigger' => [],
+                            'options'      => [
+                                'content'  => '<h3>' . esc_html__( 'Add Transactions', 'erp' ) . '</h3>' .
+                                                '<p>' . esc_html__( 'Now you are ready to have your transactions including sales, purchase and expenses.', 'erp' ) . '</p>',
+                                'position' => [ 'edge'  => 'top', 'align' => 'left' ]
+                            ],
+                        ]
+                    ];
+                    break;
+
+
+                case 'journal':
+                    $tab_pointers = [
+                        'journal' => [
+                            'target'       => '#erp-accounting .erp-journal-new',
+                            'next'         => '',
+                            'next_url'     => admin_url( $this->default_url ),
+                            'last_step'    => true,
+                            'next_trigger' => [],
+                            'last_step'    => true,
+                            'options'      => [
+                                'content'  => '<h3>' . esc_html__( 'Journal Entry', 'erp' ) . '</h3>' .
+                                                '<p>' . __( 'Journal entry is available under the Transactions as well. <br /><br />You are done and best of luck!', 'erp' ) . '</p>',
+                                'position' => [ 'edge'  => 'top', 'align' => 'left' ]
+                            ],
+                        ]
+                    ];
+                    break;
+
+            default:
+                break;
+        }
+
+        $pointers = [
+            'pointers' => $tab_pointers
+        ];
+
+        $this->enqueue_pointers( $pointers );
+	}
+
+	/**
+	 * Enqueue pointers and add script to that page.
+     *
+	 * @since 1.8.7
+     *
+	 * @param array $pointers
+     *
+     * @return void
+	 */
+	public function enqueue_pointers( $pointers ) {
+		$pointers = rawurlencode( wp_json_encode( $pointers ) );
+
+		wp_enqueue_style( 'wp-pointer' );
+		wp_enqueue_script( 'wp-pointer' );
+
+		wc_enqueue_js(
```

Comment:
On what basis this function is used here? This function is provided by WooCommerce. But we don't have the WooCommerce dependency. So it will undoubtedly throw error while WooCommerce plugin is not activated. Kindly remove the function and declare a equivalent custom function.